atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/keyvalue/impl/TableTasksTest.java

```diff
@@ -152,4 +153,23 @@ public void visit(Transaction transaction, Iterator<Cell> partialDiff) {
         Assert.assertEquals(partialRows, rowsPartiallyInCommon.get());
         Assert.assertEquals(keys1.keySet().size(), rowsVisited.get());
     }
+
+    @Test
+    public void shouldAllowSameTablenameDifferentNamespace() {
+        TableReference fooBar = TableReference.createUnsafe("foo.bar");
+        TableReference bazBar = TableReference.createUnsafe("baz.bar");
+
+        // try create table in same call
+        kvs.createTables(
+                ImmutableMap.of(
+                        fooBar, AtlasDbConstants.GENERIC_TABLE_METADATA,
+                        bazBar, AtlasDbConstants.GENERIC_TABLE_METADATA));
+
+        // try create table spanned over different calls
+        kvs.createTable(fooBar, AtlasDbConstants.GENERIC_TABLE_METADATA);
+        kvs.createTable(bazBar, AtlasDbConstants.GENERIC_TABLE_METADATA);
+
+        // Simultaneously clean up this test and ensure that tables actually were created
```

Comment:
yeah, I actually thought about this, and found another bug. I'm going to make it so all impls of dropTables actually succeed and just warn when you drop(nonExistantTable) because that will make the call idempotent.